php/libraries/Candidate.class.inc

```diff
@@ -98,7 +98,8 @@ class Candidate
 
         if (!is_array($row) || sizeof($row) == 0) {
             throw new LorisException(
-                "Could not select Candidate data from the database (DCCID: $candID)",
+                'Could not select Candidate data from the database (DCCID: '
+                                                . htmlspecialchars($candID),
```

Comment:
Would the `htmlspecialchars` break anything in this context though?

The alternative would seem to be finding every instance where the Candidate class _might_ throw an exception error and sanitizing each of those which would be difficult to do exhaustively.